UnityProject/Assets/Scripts/Player/LayDown.cs

```diff
@@ -57,7 +57,14 @@ public void OnLayDown(bool oldValue, bool newValue)
 
 		private void LayingDownLogic(bool forceState = false)
 		{
-			if (forceState) sprites.localRotation = layingDownRotation;
+			if (forceState)
+			{
+				if (sprites != null)
```

Comment:
```suggestion
			if (forceState && sprites != null)
```
combine the conditions, no need to make another level of indentation.